webapp/channels/src/components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -215,31 +267,29 @@ function createDMIntroMessage(channel: Channel, centeredIntro: string, teammate?
                 <div className='post-profile-img__container channel-intro-img'>
                     <ProfilePicture
                         src={src}
-                        size='xl'
+                        size='xl-custom-DM'
+                        status={teammate.is_bot ? '' : channel.status}
```

Comment:
I thought that as well, but wasn't getting the status icon to show up when using `teammate.status`. It seems like we use `channel.status` on the LHS for the DM as well.

Reply:
Because we have the teammate here already, we should use `teammate.status`. `channel.status` is used in the LHS, but I think the way that the status is added to the `Channel` object impacts performance, so I'd rather not use it here if we can avoid it.

The reason it might not have shown the status might have been because the user was offline and that's represented a bit differently depending on where we are in the code. This might work though
```suggestion
                        status={teammate.status ?? Constants.UserStatuses.OFFLINE}
```

Reply:
Unfortunately, that doesn't work for me either @hmhealey. That's showing the status indicator as offline no matter what if I use that code.
<img width="729" alt="image" src="https://github.com/mattermost/mattermost/assets/2040554/c88107b1-1f48-450d-a4ee-b9a628e585a2">

Reply:
This is the only outstanding issue right now as I can't seem to get a proper status value from `teammate.status`

Reply:
@hmhealey are you ok with leaving this as `channel.status` since I seem to be getting `undefined` from `teammate.status`?